packages/manager/modules/exchange/src/account/add/account-add.controller.js

```diff
@@ -223,7 +223,9 @@ export default class ExchangeAccountAddController {
       })
       .catch((error) => {
         this.messaging.writeError(
-          this.$translate.instant('exchange_ACTION_add_account_error_message'),
+          this.$translate.instant('exchange_ACTION_add_account_error_message', {
+            message: error.message,
```

Comment:
Nitpick:
```suggestion
            message: error.message | error.data.?message,
```

Reply:
Regarding this API it return `message` field as first level in `error` body object.
But, i noticed for some others API it returned into `data` object.

So, changed to be aligned with our practice.
Thanks